### PR TITLE
Update, fix install on linux

### PR DESCRIPTION
**Pull request by amenk**
* Cocoa is not available on linux
* Updated version 4.13.1
* New HEAD URL (project switched to git)
* HEAD can currently not be build because configure is missing there

see also https://github.com/Homebrew/linuxbrew/issues/488

### CODE REVIEW

**Comment by dunn** [comment]
We generally don't add things that are only for Linuxbrew, like `OS.mac?`, to the core. That does make things more complicated for Linux users, but if we tried to make Homebrew compatible with Linux by default that would require a lot of additional complexity.

**Comment by DomT4** [comment]
The Linux fix will need to stay in Linuxbrew, per our standard rules.

**Comment by sjackman** [comment]
http://bot.brew.sh/job/Homebrew%20Pull%20Requests/29802/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_gwenhywfar/
```
Error: 1 problem in 1 formula
gwenhywfar:
 * Stable: SHA1 checksums are deprecated, please use SHA256
```

**Comment by sjackman** [comment]
Please change the `sha1` to `sha256`.

File: Library/Formula/gwenhywfar.rb [hunk 1]
```diff
@@ -18,15 +18,18 @@ class Gwenhywfar < Formula
...
   option "with-check", "Run build-time check"
 
   def install
     guis = []
     guis << "gtk2" if build.with? "gtk+"
     guis << "qt4" if build.with? "qt"
-    guis << "cocoa" if build.with? "cocoa"
+    if OS.mac?
+      guis << "cocoa" if build.with? "cocoa"
+    end
```
Comment (bfontaine) [inline Library/Formula/gwenhywfar.rb]: You can do that in one line:

```ruby
guis << "cocoa" if OS.mac? && build.with? "cocoa"
```

File: Library/Formula/gwenhywfar.rb [hunk 2]
```diff
@@ -18,15 +18,18 @@ class Gwenhywfar < Formula
   depends_on "libgcrypt"
   depends_on "gtk+" => :optional
   depends_on "qt" => :optional
-
-  option "without-cocoa", "Build without cocoa support"
+  if OS.mac?
+    option "without-cocoa", "Build without cocoa support"
+  end
```
Comment (bfontaine) [inline Library/Formula/gwenhywfar.rb]: You can do that in one line:

```ruby
option "without-cocoa", "Build without cocoa support" if OS.mac?
```

**Comment by mikemcquaid** [comment]
Please make this PR just the version update and then submit a separate PR to Linuxbrew to add the `OS.mac?` stuff, thanks.

**Comment by amenk** [comment]
But why? ... to much work, sorry. But you can split it yourself if you want.

**Comment by amenk** [comment]
Ah I got now the point that you mean to make a separate PR to the other project (Linuxbrew)
Maybe I understand https://github.com/Homebrew/linuxbrew/blob/master/CONTRIBUTING.md wrong - but I was thinking that homebrew should be easy to merge, so isn't it better if the code for Linux and Mac is the same so using `OS.mac?` should be fine?

**Comment by mikemcquaid** [comment]
> so isn't it better if the code for Linux and Mac is the same so using OS.mac? should be fine?

Sorry, we don't do that in Homebrew core. Homebrew's code is OS X only.

**Comment by sjackman** [comment]
@amenk You can send the version bump to `Homebrew/hombrew` and the Mac/Linux compatibility chunks to `Homebrew/linuxbrew`.

**Comment by amenk** [comment]
updated

File: Library/Formula/gwenhywfar.rb [hunk 3]
```diff
@@ -18,7 +18,6 @@ class Gwenhywfar < Formula
   depends_on "libgcrypt"
   depends_on "gtk+" => :optional
   depends_on "qt" => :optional
-
```
Comment (mikemcquaid) [inline Library/Formula/gwenhywfar.rb]: Please don't make unnecessary changes like this, thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `gwenhywfar 4.13.1`.
